store/mongo/datastore_mongo_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/mendersoftware/go-lib-micro/identity"
```

Comment:
general comment on units: I think we should have a couple of scenarios where we explicitly verify that we do not access users, passwords, settings, tokens, of other tenants. I would just really feel more comfortable then. if I am the only one, then be it, but please consider.

Reply:
I think this is a general question if we shouldn't make a step back and think for a moment what and how we're testing;
I'm not planning to add more tests of that kind as part of this task, but I can create a QA ticket

Reply:
ok, lets create a ticket.